src/Dialect/ONNX/ONNXOps/Math/Clip.cpp

```diff
@@ -41,46 +59,80 @@ LogicalResult ONNXClipOpShapeHelper::computeShape() {
 // Shape Inference
 //===----------------------------------------------------------------------===//
 
-LogicalResult ONNXClipOp::inferShapes(
-    std::function<void(Region &)> doShapeInference) {
+template <typename T>
```

Comment:
Nit: we typically put all this in one line.

Reply:
The clang-format command divides the line to two lines. The line might be too long??